src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -184,8 +183,6 @@ private FirefoxOptions buildFirefoxOptions(TestName testName) throws IOException
                 DOM_MAX_SCRIPT_RUN_TIME, (int) getElasticTime().seconds(600));
         firefoxOptions.addPreference(
                 DOM_MAX_CHROME_SCRIPT_RUN_TIME, (int) getElasticTime().seconds(600));
-        firefoxOptions.addPreference(
-                DOM_DISABLE_BEFOREUNLOAD, false); // TODO remove when we require Firefox 129 or newer
```

Comment:
https://www.jenkins.io/doc/book/platform-information/support-policy-web-browsers/ version 129 is from last August, so seems safe to drop support for it now. We now fully rely on the officially supported BiDi technology (enabled on the next line) rather than this undocumented preference.